spec/requests/api/v1/posters_request_spec.rb

```diff
@@ -95,8 +97,26 @@
   end
 
   it "generates a new poster" do
-    post "/api/v1/posters"
+    test_params = {
+      name: "DEFEAT",
+      description: "It's too late to start now.",
+      price: 35.00,
+      year: 2023,
+      vintage: false,
+      img_url:  "https://unsplash.com/photos/brown-brick-building-with-red-car-parked-on-the-side-mMV6Y0ExyIk"
+    }
+    post "/api/v1/posters", params: test_params.to_json, headers: { "Content-Type" => "application/json" }
```

Comment:
Note: Do additional research about "application/json." This additional was fundamental to our tests passing.